source/edge-server/permissions.txt

```diff
@@ -0,0 +1,266 @@
+Or you can get more granular with rules to specify a subset of data that
+the Edge Server instance can read and write.
+
+All Edge Servers
+````````````````
+
+You can create a role that applies to all Edge Server instances by specifying
+an ``applyWhen`` :ref:`rule expression <expressions>` where the 
+:ref:`%%user_type <rule-expression-user-expansion>` is ``edge``:
+
+.. code-block:: json
+
+   {
+     "name": "edgeServerRole",
+     "apply_when": {
+       "%%user.type": {
+         "$eq": "edge"
+       }
+     },
+     ...customize the permissions for Edge Servers...
+   }
+
+For example, if your data set does not contain sensitive data, such as a 
+weather service, you might add a role stating that Edge Servers can read and 
+write all data:
+
+.. code-block:: json
+
+   {
+     "name": "readAndWriteAll",
+     "apply_when": {
+       "%%user.type": {
+         "$eq": "edge"
+       }
+     },
+     "document_filters": {
+       "read": true,
+       "write": true
+     },
+     "insert": true,
+     "delete": true,
+     "search": true,
+     "read": true,
+     "write": true
+   }
+
+A Specific Edge Server
+``````````````````````
+
+If your data set contains sensitive information that should not sync to
+every Edge Server instance, you can use permissions to filter the data so a 
+specific instance can only access a subset of the data. Or you can use 
+permissions to configure an Edge Server instance to read data, but not write
+it.
+
+Because every Edge Server instance is a user, you can use the Edge Server's
+``user_id`` to configure permissions for a given instance. You can create a 
+role that restricts the data that syncs with a specific Edge Server instance
+using a :ref:`rule expression <expressions>` where the 
+:ref:`%%user_id <rule-expression-user-expansion>` is the Edge Server's 
+``user_id``. You can get the Edge Server instance ``user_id`` from the
+:ref:`edge-server-details`, or from the Admin API. [PLACEHOLDER - Add link 
+to the relevant Admin API endpoint when the docs PR is merged.]
+
+Read/Write Own Documents
+++++++++++++++++++++++++
+
+You can restrict an Edge Server instance to read and write only its own 
+documents using the ``user_id``. For example, if an Edge Server instance 
+represents a health care clinic, you might only want to sync data relevant to 
+that clinic's patients:
+
+.. code-block:: json
+
+   {
+     "name": "facilityItemsOnly",
+     "apply_when": {},
+     "document_filters": {
+       "write": {
+         "facility_id": "%%user.id"
+       },
+       "read": {
+         "facility_id": "%%user.id"
+       },
+     },
+     "read": true,
+     "write": true,
+     "insert": true,
+     "delete": true,
+     "search": true
+   }
+
+In this example, every document has a ``facility_id`` property whose value is
+the Edge Server's ``user_id``. This role means that the Edge Server can only
+read and write documents with a ``facility_id`` matching its own ``user_id``.
+
+Read All, Write Own
```

Comment:
```suggestion
Read All, Write Own Documents
```
if we're using this format. do we want to specify these as examples: e.g. "Read/Write Own: Example"?